lib/active_model/serializer/adapter/json_api/deserialization.rb

```diff
@@ -0,0 +1,72 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module Deserialization
+          module_function
+
+          # Parse a Hash representing a JSON API document into an ActiveRecord-ready hash.
+          #
+          # @param [Hash] document
+          # @param [Hash] options
+          #   fields: Array of symbols and a Hash. Specify whitelisted fields, optionally
+          #     specifying the attribute name on the model.
+          # @return [Hash] ActiveRecord-ready hash
+          #
```

Comment:
This should be in a method header comment on the parse method (or something like it)